src/index.html

```diff
@@ -26,6 +26,32 @@
         <li><a href="#">Games</a></li>
       </ul>
     </header>
+    <section class="popup">
+      <div class="popup-container">
+        <img class="popup-img" src="" alt="">
+      <div class="popup-content">
+        <h1 class="close-btn">X</h1>
```

Comment:
Please consider using `button` instead of `h1` for the close button.

Reply:
@IndieCoderMM I will do accordingly. Thank you so much.